proposals/1984_ca-bundle-from-secret-ref/README.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "CA Bundles from secret refs"
+authors: [ "@cfryanr", "@joshuatcasey" ]
+status: "proposed"
+sponsor: [ ]
+approval_date: ""
+---
+
+*Disclaimer*: Proposals are point-in-time designs and decisions.
+Once approved and implemented, they become historical documents.
+If you are reading an old proposal, please be aware that the
+features described herein might have continued to evolve since.
+
+# CA Bundles from secret refs
+
+## Problem Statement
+
+Many Pinniped custom resources (CRs) have an inline `certificateAuthorityData` to specify a base-64 encoded CA Bundle.
+These fields require manual intervention to set and update.
+Customers who use Kubernetes-native certificate-management tooling (such as `cert-manager`) may wish to keep their
+CA bundles in a secret that their tooling can manage.
+
+## Proposed solution
+
+The users can specify a `secretName` anywhere they specify a `certificateAuthorityData`.
+Whenever `certificateAuthorityData` is empty and `secretName` is populated, Pinniped will expect a secret in the same
```

Comment:
Clarify which namespace this means. "Pinniped Supervisor and Concierge will expect a secret in their installed namespace"

Reply:
Clarified in the most recent revision. Marking this as resolved.